### PR TITLE
Support Meson build system

### PR DESCRIPTION
**Pull request by paulmenzel**
In MarIuX, make sure to have Meson installed, for example, by doing `. /pkg/python-3.6.4-0/profile`.

### CODE REVIEW

**Comment by buczek** [comment]
I suggest not to fall back to another build type if meson is not available. Instead continue and let it fail when meson is still not available when is is actually used. I'd prefer deterministic builds, which do not work around bugs in the callers environment. Also `print_warning` or `print_error` would be more noticeable in the output then `print_info`

```sh
if ! type -p meson >/dev/null 2>&1 ; then
    print_error "Could not find executable meson."
fi
```

**Comment by paulmenzel** [comment]
As discussed, I removed the check entirely, and updated the branch.

File: buildtypes/meson.sh.in [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+BEE_BUILDTYPE_DETECTED=meson
+
+BEE_BUILDTYPE_MESON_SOURCEDIR=${BEE_BUILDTYPE_MESON_MESONBUILD%/*}
+
+#### bee_configure() ##########################################################
+
+bee_configure() {
+    start_cmd meson \
+        --prefix ${PREFIX} \
+	--buildtype=plain \
```
Comment (mariux) [inline buildtypes/meson.sh.in]: indent ;)

Comment (paulmenzel) [reply buildtypes/meson.sh.in]: Thank you. Embarrassed. Fixed.

**Pull request merged**